tests/ha/iscsi_client.pm

```diff
@@ -11,12 +11,19 @@ use base 'opensusebasetest';
 use strict;
 use warnings;
 use Utils::Backends qw(is_remote_backend);
-use utils qw(zypper_call systemctl);
+use utils qw(zypper_call systemctl ping_size_check);
 use testapi;
 use hacluster;
 use version_utils qw(is_sle);
 
 sub run {
+    my $iscsi_server = get_var('USE_SUPPORT_SERVER') ? 'ns' : get_required_var('ISCSI_SERVER');
+
+    # Perform a ping size check to several hosts which need to be accessible while
+    # running this module
+    ping_size_check(testapi::host_ip());
```

Comment:
Why not to export this API "officially"? Maybe in a different PR.

Here a list of places where it is used
 - lib/network_utils.pm
 - lib/publiccloud/ssh_interactive.pm
 - lib/Remote/Lab.pm
 - tests/kiwi_images_test/validate_local_kiwi_image.pm:

Reply:
testapi belongs to a different repository (https://github.com/os-autoinst/os-autoinst), and its deployment in o3 and osd follows a different process than this repo (os-autoinst-distri-opensuse), so not as easily done I'd say. But it's not a bad idea (at least IMO)